packages/field-base/src/utils.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Please add license header to this file for consistency with the other files in this package.

Reply:
Done.